.storybook/playground/index.js

```diff
@@ -35,16 +35,16 @@ import '@wordpress/block-library/build-style/theme.css';
 import '@wordpress/format-library/build-style/style.css';
 /* eslint-enable no-restricted-syntax */
 
+// TODO: Find a way to include those fonts.
```

Comment:
It's probably easy but I don't have more time today to figure out how to include those fonts.